release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -80,6 +80,11 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-openshift-cli"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-16-oc-adm-upgrade-status-tp"]
+==== Introducing `oc adm upgrade status` command (Tech Preview)
+
+Previously, customers could retrieve information about the status of their cluster update from the web console and from the CLI. In addition to its primary focus, the `oc adm upgrade` command provided some limited information about the upgrade status. With this release, there is now a tech preview `oc adm upgrade status` command which decouples the status information from `oc adm upgrade` and compliments the UI by providing customers with specific information regarding their cluster update, including the status of the control plane and worker node updates.
```

Comment:
Update is the preferred term per our term glossary when it comes to text that's not referring to anything hardcoded as upgrade. Also, I think adding "cluster" in front makes it a little more clear, but that part is just a suggestion:
```suggestion
Previously, customers could retrieve information about the status of their cluster update from the web console and from the CLI. In addition to its primary focus, the `oc adm upgrade` command provided some limited information about the cluster update status. With this release, there is now a tech preview `oc adm upgrade status` command which decouples the status information from `oc adm upgrade` and compliments the UI by providing customers with specific information regarding their cluster update, including the status of the control plane and worker node updates.
```